examples/IRMQTTServer/IRMQTTServer.h

```diff
@@ -221,6 +223,25 @@ const uint16_t kMinUnknownSize = 2 * 10;
 //      actual a/c unit.
 #define REPLAY_DECODED_AC_MESSAGE false
 
+// ------------------------ SHT-3x Support -------------------------------------
+// Set the following to true enable SHT-3x sensor support (such as the Lolin
+// SHT30 Shield), connected to GPIOs 4 and 5 (D2 and D1).
+// *** IMPORTANT ***
+// You must also uncomment the line in the platformio.ini file to enable the
+// SHT-3x library.
+// #define SHT3X_SUPPORT true
```

Comment:
Do you need to telll people to uncomment this line too?

Reply:
Yes, I extended the comment and it's all a bit hard to read. I have rewritten.